src/components/ListItem.jsx

```diff
@@ -44,6 +44,13 @@ export function ListItem({
 		);
 	};
 
+	const handleDelete = (e) => {
+		e.preventDefault();
+		if (window.confirm('Are you sure you want to delete this item?')) {
+			deleteItem(listToken, itemId);
+		}
+	};
+
```

Comment:
This `window.confirm` method is so cool! 😁

May be a bit extra but how do you feel about displaying the item in the message to highlight exactly which item the user is intending to delete? 

ex:
```
		if (window.confirm(`Are you sure you want to delete "${name}"?`)) {
			deleteItem(listToken, itemId);
		}
```

Reply:
LOVE!